frontend/packages/dev-console/src/components/import/icon/IconDropdown.scss

```diff
@@ -1,6 +1,16 @@
 .odc-icon-dropdown {
   img.icon {
+    background-color: var(--pf-v5-global--palette--white);
+    border-radius: var(--pf-v5-global--BorderRadius--sm);
+    height: var(--pf-v5-global--FontSize--4xl);
     margin-right: var(--pf-v5-global--spacer--sm);
+    padding: var(--pf-v5-global--spacer--xs);
+    width: var(--pf-v5-global--FontSize--4xl);
```

Comment:
I agree that 3xl looks nicer too--its just that my laptop screen is a bit small so I couldn't really see the icons very well

Reply:
Can you change the height and width to 3xl?

Reply:
Is 3xl equivalent to 16px? PF specs menu items at 16px so I recommend we follow those guidelines.
![image](https://github.com/user-attachments/assets/1bb303d0-6d7d-43e3-a811-24273efc40e5)

Reply:
Hey! This is the dropdown with the width and height set at 16px:

![image](https://github.com/user-attachments/assets/7d1363be-9095-492a-adfa-f32e068b0bc6)

I'm personally leaning towards using the 3xl (1.75rem) size, since at 16px the icons are a bit hard to differentiate. What do you think?